tests/emukit/conftest.py

```diff
@@ -0,0 +1,38 @@
+import GPy
+import numpy as np
+import pytest
+
+from emukit.core import ContinuousParameter, ParameterSpace
+from emukit.model_wrappers import GPyModelWrapper
+from emukit.model_wrappers.gpy_quadrature_wrappers import convert_gpy_model_to_emukit_model
+from emukit.quadrature.methods import VanillaBayesianQuadrature
+
+
+@pytest.fixture
+def n_dims():
+    # default to 2 dimensional inputs for tests, but can be overridden by individual tests if desired
```

Comment:
How to override this in a test? Define a fixture with the same name?

Reply:
Added a comment on one way to do it. There are others, one of which is to override the fixture in the local file with the same name.